### PR TITLE
Connect temp db to

### LINKED ISSUE #13: Connect temp DB

### PR DESCRIPTION
**Pull request by laska-sl**
Added DataContext class, User and CelebrationDate models, added default connection string, added DbContext to Startup

### CODE REVIEW

**Comment by sponkrashin** [comment]
Why do you have commit from another PR here?

**Review by sponkrashin — CHANGES_REQUESTED** [review]
Your DB models differ from what was discussed earlier. At least, you won't store separate celebrating dates in the database. I suggest you to add one test model with structure, create test data table and check whether it works.
Also DB models naming should be "...Entity".

**Comment by laska-sl** [comment]
> Your DB models differ from what was discussed earlier. At least, you won't store separate celebrating dates in the database. I suggest you to add one test model with structure, create test data table and check whether it works.
> Also DB models naming should be "...Entity".

Decided to defer models for later. They are deleted

File: ArcadiaParties.API/Startup.cs [hunk 1]
```diff
@@ -39,6 +49,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            //app.UseHttpsRedirection();
```
Comment (sponkrashin) [inline ArcadiaParties.API/Startup.cs:52]: Removed it instead of comment.

File: ArcadiaParties.API/Startup.cs [hunk 2]
```diff
@@ -23,6 +31,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            //temp SQLite connection
+            services.AddDbContext<DataContext>(x => x.UseSqlite(Configuration.GetConnectionString("DefaultConnection")));
             services.AddSwaggerGen(c =>
```
Comment (sponkrashin) [inline ArcadiaParties.API/Startup.cs:36]: Add an empty line after separate logical block.

File: ArcadiaParties.API/Startup.cs [hunk 3]
```diff
@@ -23,6 +31,8 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
 
+            //temp SQLite connection
```
Comment (sponkrashin) [inline ArcadiaParties.API/Startup.cs:34]: Why "temp"? If you are going to change it later, why don't do this now? If not - why "temp"?

File: ArcadiaParties.API/appsettings.json [hunk 4]
```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=ArcadiaParties.db"
```
Comment (sponkrashin) [inline ArcadiaParties.API/appsettings.json:3]: Where is a file itself?

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties.API/appsettings.json (hunk 4 above)
Comment (laska-sl) [reply ArcadiaParties.API/appsettings.json:3]: Cause we will use seeding data, DB file creates at run project.

Comment (sponkrashin) [reply ArcadiaParties.API/appsettings.json:3]: Ok.

File: ArcadiaParties.API/Startup.cs (hunk 1 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs:52]: Line removed. Watch new "Code style changes" PR

File: ArcadiaParties.API/Startup.cs (hunk 2 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs:36]: Empty line added. Watch new "Code style changes" PR

File: ArcadiaParties.API/Startup.cs (hunk 1 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs]: Line removed. . Watch new PR

File: ArcadiaParties.API/Startup.cs (hunk 3 above)
Comment (laska-sl) [reply ArcadiaParties.API/Startup.cs:34]: Comment changed.  Watch new "Code style changes" PR

**Pull request merged**